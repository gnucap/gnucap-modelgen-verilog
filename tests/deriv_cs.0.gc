
attach ../modelgen_0.so

demo_module test_cs0(np, nn);

analog begin
	real i;
	i = 1;
	I(np,nn) <+ i;
end

endmodule

!make test_cs0.so

attach ./test_cs0.so

verilog

test_cs0 dut(1,2);
vsource v1(1, 0);
vsource #(.dc(1)) v2(2, 0);

list

print dc i0(dut) v0(dut) v1(dut)
dc v1 0 1 .1
