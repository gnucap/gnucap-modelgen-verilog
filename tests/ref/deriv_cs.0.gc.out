g++ -DTRACE_UNTESTED -g -O0 -fPIC  -I../include test_cs0.cc -o test_cs0.o -c
g++ -shared  -I../include test_cs0.o  -o test_cs0.so
rm test_cs0.o

demo #() dut ();
vsource #() v1 (.p(1),.n(0));
vsource #(.dc(1)) v2 (.p(2),.n(0));
#           i0(dut)    v0(dut)    v1(dut)   
 0.         1.         0.         1.        
 0.1        1.         0.1        1.        
 0.2        1.         0.2        1.        
 0.3        1.         0.3        1.        
 0.4        1.         0.4        1.        
 0.5        1.         0.5        1.        
 0.6        1.         0.6        1.        
 0.7        1.         0.7        1.        
 0.8        1.         0.8        1.        
 0.9        1.         0.9        1.        
 1.         1.         1.         1.        
